.github/workflows/nightly-tests.yml

```diff
@@ -14,7 +14,7 @@ on:
           - dev
         default: 'perf'
   schedule:
-    - cron: "0 6 * * *" # runs at 6 AM UTC, 1 AM EST.
+    - cron: "0 6 * * *" # runs at 6 AM UTC, 2 AM EST.
```

Comment:
1am is correct for EST (UTC-05). 2am is EDT.

Reply:
Maybe I'm missing something: https://www.google.com/search?q=6+am+utc+to+est

Reply:
Interesting, although EST is still UTC-5 and EDT is UTC-4. When I use that link it mentions ET (aka Eastern Time) but that's something else.

We were so close to going DST all the time in 2023 but it looks like that's not going to happen anytime soon. https://thehill.com/homenews/house/3571007-permanent-daylight-saving-time-hits-brick-wall-in-house/

Anyways I still think UTC-5 is correct for EST. If you like you can remove the EST from the comment completely since UTC alone is sufficient and doesn't change twice a year.

Reply:
See https://en.wikipedia.org/wiki/Eastern_Time_Zone